Unity3D/Assets/RosSharp/Scripts/OdometrySubscriber.cs

```diff
@@ -15,7 +15,8 @@
 limitations under the License.
 */
 
-
+// Topic is now public, allowing editing (C) Federal Univerity of Minas Gerais (Brazil), 2017
```

Comment:
please change your comment to:
+// Made subscription topic a public variable. Federal Univerity of Minas Gerais (Brazil), 2017